### PR TITLE
Add deploying pipeline

### PR DESCRIPTION
**Pull request by acidicMercury8**
Changes proposed in this pull request:

- Added deploying pipeline

**Warning**! It's also need to create `VSCE_PAT` secret in the repository settings!

Related issue:

- #5 

Agreements

- [x] I agree to follow this project's [__Contributing Guidelines__](https://github.com/hexalang/.github/blob/kawaii/CONTRIBUTING.md)
- [x] I agree to follow this project's [__Code of Conduct__](https://github.com/hexalang/.github/blob/kawaii/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: .github/workflows/building-bundle.yml [hunk 1]
```diff
@@ -1,6 +1,13 @@
 name: Building bundle
 
-on: [ push, pull_request ]
+on:
+  push:
+    branches: [ '**' ]
```
Comment (PeyTy) [inline .github/workflows/building-bundle.yml]: Seems unneeded

File: .github/workflows/building-bundle.yml [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
+  pull_request:
+    branches: [ '**' ]
```
Comment (PeyTy) [inline .github/workflows/building-bundle.yml]: Seems unneeded

File: .github/workflows/building-bundle.yml (hunk 1 above)
Comment (acidicMercury8) [reply .github/workflows/building-bundle.yml]: @PeyTy I've already talked about this

File: .github/workflows/building-bundle.yml [hunk 3]
```diff
@@ -34,3 +39,33 @@ jobs:
...
+        fetch-depth: 0
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+
+    - name: Publish Hexa VSCode bundle
+      if: startsWith(github.ref, 'refs/tags/releases/')
```
Comment (PeyTy) [inline .github/workflows/building-bundle.yml:67]: Confirming that this works as intended

Comment (acidicMercury8) [reply .github/workflows/building-bundle.yml:67]: @PeyTy this option is recommended in the extension deployment documentation

**Pull request merged**